samples/Common.c

```diff
@@ -361,14 +362,37 @@ STATUS initializePeerConnection(PSampleConfiguration pSampleConfiguration, PRtcP
             }
         }
     }
-
+    pSampleConfiguration->iceUriCount = uriCount + 1;
```

Comment:
Not sure why we have this - uriCount is an invariant. Is the intent to change this later?

Reply:
I am using pSampleConfiguration->iceUriCount while pulling stats for all the servers. That is the reason I am storing uriCount in pSampleConfiguration.

Reply:
I guess I am not understanding the reason of setting uriCount to 0 then assigning it to pSampleConfiguration->iceUriCount + 1; Perhaps, we will be mutating this value later? If not, why not simply assign 1?

Reply:
> Ah! Ok. I assigned it to 0 to be consistent with the rest of the logic in the function. Because we do a uriCount + 1 within the for loops. If I set it to 1 outside, the value inside the for loops would be incorrect. And we do mutate it inside the loops